### DISCUSSION TITLE
API Metadata stripped?

### DISCUSSION THREAD

**Discussion by dumdumdumdum** [tommyblue/smugmug-backup#54] [Q&A]
First, thank you for creating this..haven't downloaded yet. 
Recently read that Smugmug is stripping metadata from APIs now. Can anyone confirm if photo downloads from smugmug-backup seem to be missing creation dates, for example?

**Comment by tommyblue** [comment]
where did you read that?

**Reply by dumdumdumdum** [reply]
Unsure.... Dgrin? Reddit? Some other photo forum? (I'd have to spend some time back-tracking.) I don't see any update on SM's API page. If you haven't heard or noticed anything (and/or feel creation-date metadata is preserved), that's enough for me!

**Reply by tommyblue** [reply]
I didn't notice anything different, but honestly my daily backup isn't actively monitored, I'll take a look but from a first look I didn't notice anything. Since I'm working on a new version of the software, based on Electron, I think I'll have chances to dig it

**Comment by dumdumdumdum** [comment]
No worries. I’ve been following you since you initiated your project.…don’t self-host but considering. Truly interested in a Smugmug solution like yours.On Aug 11, 2024, at 12:32 PM, Tommaso Visconti ***@***.***> wrote:﻿
I didn't notice anything different, but honestly my daily backup isn't actively monitored, I'll take a look but from a first look I didn't notice anything. Since I'm working on a new version of the software, based on Electron, I think I'll have chances to dig it
